### ISSUE TITLE
[Feature] Does Qwen2-VL support W4A16 in turbomind engine?

### ISSUE THREAD

**Issue by BlueBlueFF** [InternLM/lmdeploy#2623]
### Motivation

Does Qwen2-VL support W4A16 in turbomind engine?

### Related resources

Does Qwen2-VL support W4A16 in turbomind engine?

### Additional context

Does Qwen2-VL support W4A16 in turbomind engine?

**Comment by lvhan028** [comment]
No. Neither qwen2-vl or qwen2-vl-w4a16 is supported turbomind engine.
Please try lmdeploy's pytorch engine
